folly/concurrency/ConcurrentHashMap.h

```diff
@@ -30,7 +30,7 @@ namespace folly {
  * support erase and update.
  *
  * Readers are always wait-free.
- * Writers are sharded, but take a lock.
+ * Writers are shared, but take a lock.
```

Comment:
This was correct as it was. Writers take a lock, but only lock part of the map, so are sharded.